### ISSUE TITLE
Review and test new social media content planning apps

### ISSUE THREAD

**Issue by clara-zi** [Giveth/giveth-planning#1170]
Aim: 
To have all our social content planning (text, media) happening in one app. with calendar view, review and approval process ‘in-app.’ to improve process flow, align social content with the purpose of working more efficiently, more effectively and freeing up time to work on engagement across platforms.

One of the most significant challenges is finding an app. that provides all the functionality of ‘Tweet Hunter’: the planning app. currently used for Twitter content, which is a powerful tool and arguable the best on the market for tweet planning. These are the key features that make Tweet Hunter difficult to beat: 

•	Functionality to plan Tweet threads
•	Detailed Twitter analytics
•	Best Time to Tweet 
•	AI Tweet Generator

Other important desired features
•	A simple and effective content review/approve across all platforms in a single app.

**Comment by clara-zi** [comment]
Comparison of 2 best shortlisted apps: Content Studio, Vista Social.

https://docs.google.com/document/d/1BKkgQgG8ni7m8z6tqWkCnmFLsDlyn1lrjSNG4bhTf9I/edit?usp=sharing
@c0ric0ri @SymSL

**Comment by clara-zi** [comment]
@c0ric0ri wdyt? suggest 'live' testing content planning for FB, LI and Twitter using 'calendar view' and tools available with Content Studio (2-week free trial).

Suggest maintaining all current Twitter planning, review and posting process in parallel.

Establish and tweak, content planning, review, approval, scheduling and posting through Content Studio for FB and LI as full test to ensure it ticks all boxes.

**Comment by clara-zi** [comment]
@c0ric0ri @SymSL Content Studio set-up and connected to FB and LI - free trial.
Draft post created as test for review / approval process. Suggest we use the 'comment' function for feed-back / edit suggestions. Once edits agreed, required updates can be put in 'task' and checked off as post goes to final draft.
Free trial limits the number of team members that we are allowed to register.
@c0ric0ri it would be great if you could go into the app. and play about with review and approval, before the end of the week, so we can make the most of the free trial. Thanks.

**Comment by clara-zi** [comment]
@SymSL let me know if you need access to take a look around the app.

**Comment by clara-zi** [comment]
some key benefits, noting here as I find them, include: 
- less entries on the Github board making it easier and more efficient to deal with 'non-social' issues
- it will mean less backwards and forwards between apps and documents
- less typing, linking & inter-webbing of issues on GitHub 
- better alignment of scheduling across apps
- use of AI for content generation for all platforms
- more efficiencies of media preparation (no duplication and copying across from 1 doc to the other)
- ease of assigning resp. to each contributor,
- a visual calendar planner
- clear visual deadlines
- cool feature to help build engagement and works across all platforms: an 'inbox' that detects any activity on the platform that is related to Giveth where you can assign someone to respond/follow-up, track this activity and mark as 'complete' when done.
- the AI content generator is pretty good
- best time of day to post 
- suggests hashtags
- has a good suite of analytics
- AI content and caption generation
- Tweet threads

**Comment by clara-zi** [comment]
@c0ric0ri please test app. for Twitter planning as a comparison to Tweet Hunter. Tx

**Comment by clara-zi** [comment]
Thanks @c0ric0ri for testing reviewing capabilities of CS.

Option 1 - reviewing/editing done in app.
Commenting and task allocation against alongside posts possible and provides good functionality but "It's just hard to beat the suggesting mode of Google Docs where you can literally fix typos or change language mid-sentence or move things around".

Option 2
Link to Google doc (for text only) can be added to post placeholder or added in comments, where reviewing and editing can take place. 
Image/design of post in app is superior to Google docs. as it can be applied across all platforms at once, saving duplication of effort.

**Comment by clara-zi** [comment]
Twitter review still to do @laurenluz @c0ric0ri

**Comment by c0ric0ri** [comment]
@clara-zi is there any possibility to allow Lauren to test Content Studio again?

**Comment by clara-zi** [comment]
Yes. We will need to set-up a new trial account - perhaps we could use a comms giveth email? wdyt? and then this could also be the universal super admin / account holder

**Comment by clara-zi** [comment]
Review complete, much discussion had. @c0ric0ri @laurenluz Close this issue?
